apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -144,6 +158,13 @@ export class LemonadePipeline implements BasePipeline {
     this.zupassPublicKey = zupassPublicKey;
     this.loaded = false;
 
+    this.semaphoreGroupProvider = new SemaphoreGroupProvider(
```

Comment:
should this be `undefined` if group membership option is not specified for this pipeline?